apps/allocations/app/components/Card/Budget.js

```diff
@@ -151,6 +173,20 @@ const StyledCard = styled(Card)`
   width: auto;
 `
 
+const CardBottom = styled.div`
+  padding: 4px 12px;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  position: absolute;
```

Comment:
Why do we need this? `position: absolute` can cause weird visual bugs, and I'd like to avoid it if we can!

Reply:
This positions the period info and active/inactive label at the bottom of the card. There are other ways of doing this, but I believe they'll take more lines, and rearrange other divs. Maybe a better place to adjust this would be on this PR: https://github.com/AutarkLabs/open-enterprise/pull/1761, since I will have to update that one to account for any visual changes on this PR?